src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1515,7 +1515,7 @@ function ($a) {
                                 ->addDefaultsIfNotSet()
                                 ->children()
                                     ->enumNode('default_middleware')
-                                        ->values([true, false, 'allow_no_handlers'])
+                                        ->values([true, false, 'allow_no_handlers', 'throw_no_routing'])
```

Comment:
Why not adding as a proper boolean node under `buses`, since these values are mutually exclusive?
That would allow the option to also be used as arg when using the `SendMessageMiddleware` without the defaults.